generators/generator-base.js

```diff
@@ -263,7 +263,8 @@ module.exports = class extends PrivateBase {
             let importStatement = `|import { ${appName}${entityAngularName}Module } from './${entityFolderName}/${entityFileName}.module';`;
             if (importStatement.length > constants.LINE_LENGTH) {
                 importStatement =
-                    `|import {
+                    `|//prettier-ignore
```

Comment:
add a space `// prettier-ignore`

Reply:
Since this is added please revert changes to the entity names in Travis as some of them were made longer on purpose as we do some specific thing when names are longer for some DB